aws-xray-recorder-sdk-sql/src/main/java/com/amazonaws/xray/sql/SqlSubsegments.java

```diff
@@ -18,14 +18,12 @@
 import com.amazonaws.xray.AWSXRay;
 import com.amazonaws.xray.entities.Namespace;
 import com.amazonaws.xray.entities.Subsegment;
-
```

Comment:
These formatting changes were failing the build, although I don't agree with them, added them for now and we can revisit the linter later.